tests/publiccloud/validate_repos.pm

```diff
@@ -44,9 +44,6 @@ sub run {
     }
 }
 
-sub post_fail_hook {
```

Comment:
I would assume this was done on purpose , in case validate_repos fail we don't want to do any usual actions hence we created empty post_fail_hook to overwrite parent one. Instead of deleting it please add comment explaining this

Reply:
Thanks! The validate_repos.pm use base consoletest, anyway, so I removed this change.